Gems/Multiplayer/Code/Source/Debug/MultiplayerConnectionViewportMessageSystemComponent.cpp

```diff
@@ -186,12 +186,17 @@ namespace Multiplayer
 
     void MultiplayerConnectionViewportMessageSystemComponent::OnEditorConnectionAttempt(uint16_t connectionAttempts)
     {
-        m_centerViewportDebugText = AZStd::fixed_string<128>::format(OnEditorConnectionAttemptMessage, connectionAttempts);
+        m_centerViewportDebugText = AZStd::fixed_string<MaxMessageLength>::format(OnEditorConnectionAttemptMessage, connectionAttempts);
+    }
+
+    void MultiplayerConnectionViewportMessageSystemComponent::OnAllConnectionAttemptsFailed()
```

Comment:
nit: OnEditorConnectionFailed ? to match OnEditorConnectionAttempt

Reply:
<3 fixed!